openapi_python_client/__init__.py

```diff
@@ -127,27 +124,10 @@ def build(self) -> Sequence[GeneratorError]:
         self._create_package()
         self._build_metadata()
         self._build_dlt_config()
-        # self._build_models()
         self._build_security()
-        self._build_api()
         self._build_source()
         self._build_pipeline()
         self._run_post_hooks()
-        return self._get_errors()
-
-    def update(self) -> Sequence[GeneratorError]:
-        """Update an existing project"""
-
-        if not self.package_dir.is_dir():
-            return [GeneratorError(detail=f"Directory {self.package_dir} not found")]
-        print(f"Updating {self.package_name}")
-        shutil.rmtree(self.package_dir)
```

Comment:
you removed update but what did update meant? they are removing the full tree anyway

Reply:
Yeah exactly it's the same as deleting the folder and running init again, so I didn't really see the point in having it. At least such command should not be called "update", I would expect that to do something different (like add new endpoints or something)